.github/workflows/cypress.yml

```diff
@@ -33,6 +33,16 @@ jobs:
         with:
           node-version: ${{ steps.get-node-version.outputs.NODE_VERSION }}
 
+      - name: Cache dependencies
+        id: cache-dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/yarn
+            ~/.cache/Cypress
```

Comment:
Does adding this force the cache to update every time?

Reply:
From what I can tell, missing this hasn't affected the install time much if at all. Without including this in the branch where I consolidated everything into the CI workflow, the install during the Cypress jobs still only takes about 15-25 seconds.